docs/src/languages/solidity.md

```diff
@@ -0,0 +1,4 @@
+# Solidity
+
+- Tree Sitter: [tree-sitter-solidity](https://github.com/parmanuxyz/tree-sitter-solidity)
+- Language Server: [solidity](https://github.com/nwolverson/solidity-language-server)
```

Comment:
Is this link for the LSP repo accurate?

Reply:
Good catch, that was a leftover from copy-pasting another language's markdown file. Just pushed a commit with the correct link. Thanks @stevennevins

Reply:
Nice, the one you linked is the LSP I used to use. Have you tried this [one](https://github.com/NomicFoundation/hardhat-vscode/tree/development/server)?

Reply:
@stevennevins Not familiar with that one until now. Will give it a shot and push a commit shortly.

Reply:
Okay just switched to Nomic's language server, rebuilt the binary, and tested again. Looks good! Thanks for the tip 👍

Reply:
Got it, appreciate the feedback. I am coming back to Solidity after some time, so it's helpful to know what the latest preferred tooling is.